docs/connectors/sqlserver.asciidoc

```diff
@@ -6,18 +6,6 @@
 :icons: font
 :source-highlighter: highlight.js
 
-[NOTE]
```

Comment:
Can we leave in this note until 0.9.0.Final is out?

Reply:
@gunnarmorling  Well I exppected this to be merged with final version - hence I took a steap ahead

Reply:
Ok, but why not merge this PR for the CR1 release? I.e. why hold it back?